### ISSUE TITLE
Wrong results for Aggregations using sumSeries

### ISSUE THREAD

**Issue by Antalis** [alexanderzobnin/grafana-zabbix#511]
### I'm submitting a ...
- [X] Bug report

**Software Versions**

| Grafana | Zabbix | Grafana-Zabbix Plugin |
| - | - | - |
| 4.6.3 | 2.2.1 | 3.8.1 |

#### What did you do?
Upgrade Grafana and Grafana Zabbix  to latest versions.

#### What was the expected result?
A correct result with sumSeries

#### What happened instead?
the calculation of the sum of the series is wrong

![grafana](https://user-images.githubusercontent.com/16205746/34983241-2fd0e70e-faad-11e7-97bc-bfc1c0c265c6.jpeg)
![grafana2](https://user-images.githubusercontent.com/16205746/34983259-39151092-faad-11e7-9ff6-f9bfe07a47b7.jpeg)

With 

| Grafana | Zabbix | Grafana-Zabbix Plugin |
| - | - | - |
| 4.2.0 | 2.2.1 | 3.4.0 |

the calculation of the sum of the series is true

**Comment by alexanderzobnin** [comment]
It's due to reason described in #320 . Try to use `groupBy()` with interval which is more than metric collecting interval (like `2h` if metric is collecting every hour). This aligns points at the end of the time range.

**Comment by alexanderzobnin** [comment]
Did you try my solution?

**Comment by Antalis** [comment]
To avoid making mistakes, can you tell me what I need to put in place of  "sumSeries ()"

**Comment by alexanderzobnin** [comment]
It should be something like this, but use groupBy interval more than _Total disk space_ collecting interval (try `1d` or, maybe `1h`).

![screenshot from 2018-01-30 15-35-23](https://user-images.githubusercontent.com/4932851/35566655-391167c2-05d3-11e8-9d25-d1b5d309596d.png)

**Comment by mchtech** [comment]
I encountered the same problem after the grafana was upgraded to version 5. The calculation results of the "sumseries" are wrong. (It seems that the result is only the sum of a part of maching items)

Finally, I solved the problem **to some extent** by using the following steps:
1. Create a new DataSource with MySQL type
**2. Modify the current "Zabbix" DataSource, enable "Direct DB Connection" option, and choose the mysql data source that created in step 1.**

The probability of displaying error "sumseries" results was significantly reduced.

@Antalis @alexanderzobnin
